File: app/src/main/java/com/workshop/universityannouncementsboard/extra/Weather.java
package com.workshop.universityannouncementsboard.extra;

import android.graphics.Color;

public class Weather {

    public void updateWeather(int degrees) {
        String description;
        int color;
        if (degrees < 5) {
            description = "cold";
            color = Color.BLUE;
        } else if (degrees < 23) {
            description = "mild";
            color = Color.YELLOW;
        } else {
            description = "hot";
            color = Color.RED;
        }
        // ...
    }
}

File: app/src/main/java/com/workshop/universityannouncementsboard/extra/Weather.kt
package com.workshop.universityannouncementsboard.extra

import android.graphics.*

class Weather {

    fun updateWeather(degrees: Int) {
        val (description, color) = when {
            degrees < 5 -> Pair("cold", Color.BLUE)
            degrees < 23 -> "mild" to Color.YELLOW
            else -> "hot" to Color.RED
        }
        // ...
    }
}